packages/block-editor/src/components/block-list-appender/index.js

```diff
@@ -16,6 +17,9 @@ import { getDefaultBlockName } from '@wordpress/blocks';
 import DefaultBlockAppender from '../default-block-appender';
 import ButtonBlockAppender from '../button-block-appender';
 
+// A Context to store the map of the appender map.
+export const AppenderContext = createContext();
```

Comment:
This seems pretty similar to the way `BlockNodes` context works.

If we decide to go this way it could be worth making the code similar and also naming it `AppenderNodes` for consistency.

Reply:
Yeah, I have noticed that too! Renaming sounds good to me.